pkg/csi/cinder/driver.go

```diff
@@ -33,13 +34,13 @@ const (
 )
 
 var (
-	version = "1.2.0"
+	specVersion = "1.2.0"
```

Comment:
@jichenjc as mentioned before , this should be driver version . you can add one more variable for spec version